aspnetcore/security/authentication/social/facebook-logins.md

```diff
@@ -16,43 +16,48 @@ uid: security/authentication/facebook-logins
 
-By [Rick Anderson](https://twitter.com/RickAndMSFT), [Pranav Rastogi](https://github.com/rustd), and [Valeriy Novytskyy](https://github.com/01binary)
+By [Rick Anderson](https://twitter.com/RickAndMSFT), [Pranav Rastogi](https://github.com/rustd), [Valeriy Novytskyy](https://github.com/01binary)
 
-This tutorial shows you how to enable your users to sign in with their Facebook account using a sample ASP.NET Core project created on the [previous page](index.md). We start by creating a Facebook AppId by following the [official steps](https://developers.facebook.com/docs/apps/register).
+This tutorial shows you how to enable your users to sign in with their Facebook account using a sample ASP.NET Core project created on the [previous page](index.md). We start by creating a Facebook App ID by following the [official steps](https://developers.facebook.com/docs/apps/register).
 
 ## Creating the app in Facebook
 
-*  Navigate to [https://developers.facebook.com/apps](https://developers.facebook.com/apps) and sign in. If you don't already have a Facebook account, use the **sign up** link on the login page to create one.
+*  Navigate to the [Facebook for Developers](https://developers.facebook.com/apps) page and sign in. If you don't already have a Facebook account, use the **Sign up for Facebook** link on the login page to create one.
 
-* Tap **My Apps** in the upper right corner:
+* Click the **Create a New App** button in the upper right corner to create a new app ID.
 
 ![Facebook for developers portal open in Microsoft Edge](index/_static/FBMyApps.png)
 
-* Tap **+ Add a New App** and fill out the form to create a new app ID:
+* Fill out the form and click the **Create App ID** button.
 
 ![Create a New App ID form](index/_static/FBNewAppId.png)
 
-* The **Product Setup** page is displayed, letting you select the features for your new app. Tap **Get Started** on **Facebook Login**:
+* The **Product Setup** page is displayed, letting you select the features for your new app. Click **Get Started** on **Facebook Login**.
 
 ![Product Setup page](index/_static/FBProductSetup.png)
 
-* You are presented with the **Client OAuth Settings** page, with some defaults already set:
+* Next, a quick start process begins at a **Choose a Platform** screen. This will help you set up client-side login integration, which we won't be doing in this tutorial. 
+
+    To bypass this, click the **Settings** link in the menu at the left.
+
+
+* You are presented with the **Client OAuth Settings** page, with some defaults already set.
 
 ![Client OAuth Settings page](index/_static/FBOAuthSetup.png)
 
-* Enter your base URI with *signin-facebook* appended into the **Valid OAuth Redirect URIs** field. For example, `https://localhost:44320/signin-facebook`.
+* Enter your base URI with *signin-facebook* appended into the **Valid OAuth Redirect URIs** field. For example, `https://localhost:44320/signin-facebook`. Then, click **Save Changes**.
   
```

Comment:
Also ...

> For example, `https://localhost:44320/signin-facebook`.

... isn't a sentence. It could possibly go into parentheses ...

> Enter your base URI with *signin-facebook* appended into the **Valid OAuth Redirect URIs** field (Example:  `https://localhost:44320/signin-facebook`).